pyphare/pyphare/pharesee/plotting.py

```diff
@@ -277,6 +278,17 @@ def finest_field_plot(run_path, qty, **kwargs):
         vmax = kwargs.get("vmax", np.nanmax(DATA))
         cmap = kwargs.get("cmap", 'Spectral_r')
 
+        dx = finest_coords[0][1] - finest_coords[0][0]
```

Comment:
same comment as above : this would deserve to be in the object. also because we could have \x[1]-x[0]=0.2` while `x[2]-x[1]=0.1999999999999`